examples/ysz_experiments.jl

```diff
@@ -508,7 +447,7 @@ function run_new(;test=false, print_bool=false, debug_print_bool=false, verbose=
             # for faster computation, solving of "dtstep problem" is not performed
             U0 .= inival
             inival.=U
-            Qb0 = - integrate(sys,label_ysz_model.reaction!,U0) # \int n^F
+            Qb0 = - integrate(sys,model_symbol.reaction!,U0) # \int n^F
             dphi0_end = U0[iphi, end] - U0[iphi, end-1]
             dphiB0 = parameters.eps0*(1+parameters.chi)*(dphi0_end/dx_end)
```

Comment:
Would it be possible to use meas_tran(), and meas_std() here, since these functions are what is being evaluted here.

Reply:
yes, it is the near future plan.